docs/source/user-guide/sql/select.md

```diff
@@ -306,6 +306,8 @@ SELECT age, person FROM table ORDER BY age, person DESC;
 
 Limits the number of rows to be a maximum of `count` rows. `count` should be a non-negative integer.
 
+DataFusion does not currently support the SQL `FETCH` clause. Use `LIMIT` to constrain the number of returned rows instead.
```

Comment:
Personally I'm of the mind to omit this from documentation; I prefer to state only what we do support instead of enumerating what we don't.